cypress/integration/write-demo.spec.js

```diff
@@ -35,19 +35,41 @@ describe('Write Demo', () => {
     })
   })
 
-  describe('Happy path', () => {
-    beforeEach(() => {
-      cy.get('[data-cy=username]').type('root')
-      cy.get('[data-cy=password]').type('password')
-      cy.get('[data-cy=login]').click()
-    })
+  it.skip('Shows login failed message after failed login', () => {
```

Comment:
skipped?

Reply:
was putting them as placeholders in anticipation of API integration testing being completed https://digicatapult.atlassian.net/browse/DSBD-34 

but I can remove, or change them to not be skips (or actually write get the API calls working in the tests!)